modules/genomics_cromwell/README.md

```diff
@@ -0,0 +1,35 @@
+# RAD Lab Genomics-Cromwell Module
```

Comment:
As per [CONTRIBUTING.md](https://github.com/GoogleCloudPlatform/rad-lab/blob/main/docs/CONTRIBUTING.md#repository-structure), please add below 2 tags at the end of the README.md for populating structured Variables and Outputs automatically via [tfdoc.py](https://github.com/GoogleCloudPlatform/rad-lab/blob/main/tools/tfdoc.py).

```
<!-- BEGIN TFDOC -->
<!-- END TFDOC -->
```

Reply:
@guptamukul-google Do I need to run the tfdoc.py or will this be done automatically when merged?

Reply:
It should run automatically when merged but if you want you can run too.